src/rpc/mod.rs

```diff
@@ -355,28 +356,23 @@ impl Connection {
                 }
             }
         }
-        if let Err(err) = read_stream.shutdown(Shutdown::Read) {
-            warn!(
-                "Connection::process_reading_data fail to close read stream to {}, err:{}",
-                addr, err
-            );
-        }
+        // TODO: stream close
+        drop(read_stream);
```

Comment:
Why not shutdown like write_stream?

Reply:
No need to drop I think, when the function end, it will be dropped automatically.
In fact this is the classic RAII.